all_test.go

```diff
@@ -80,6 +100,17 @@ func TestCopy(t *testing.T) {
 		Expect(t, info.Mode()&os.ModeSymlink).Not().ToBe(0)
 	})
 
+	When(t, "copying symbolic link to a directory that doesn't exist yet", func(t *testing.T) {
+		err = Copy("testdata/case03/case01", "testdata.copy/case03/case01.2")
```

Comment:
I've just a bit confused.
Since `testdata.copy/case03` is supposed to exist here, is this case testing what we want to test?

Reply:
I can understand what you want to implement from your source code, but this test case is not that clear for me to understand the same thing.

It might be better to create another test case (`case10`?), otherwise make this test case more descriptive.
Let me know your thought. Thank you.

Reply:
I've changed it into a case10 and reworded to clarify that it ensures intermediate directories are created when copying symlinks.